packages/context/src/objects/objects.ts

```diff
@@ -62,6 +62,11 @@ export type Obj = {
 
 export type ObjWithRoot = Obj & { rootObjectId: string };
 
+export type EffectiveObject = Obj & {
```

Comment:
Why is this needed?  I don't think there's any api that returns this.

Reply:
you are right. this is only used locally in Configuration repo to buildEffectiveObjects.